microprofile/opentracing-smallrye/src/main/java/org/wildfly/extension/microprofile/opentracing/smallrye/DependencyProcessor.java

```diff
@@ -0,0 +1,36 @@
+package org.wildfly.extension.microprofile.opentracing.smallrye;
+
+import org.jboss.as.server.deployment.Attachments;
+import org.jboss.as.server.deployment.DeploymentPhaseContext;
+import org.jboss.as.server.deployment.DeploymentUnit;
+import org.jboss.as.server.deployment.DeploymentUnitProcessor;
+import org.jboss.as.server.deployment.Phase;
+import org.jboss.as.server.deployment.module.ModuleDependency;
+import org.jboss.as.server.deployment.module.ModuleSpecification;
+import org.jboss.modules.Module;
+import org.jboss.modules.ModuleIdentifier;
+import org.jboss.modules.ModuleLoader;
+
+public class DependencyProcessor implements DeploymentUnitProcessor {
```

Comment:
Nitpick: Other subsystems prefix the subsystem name, so something like OpenTracingDependencyProcessor

Reply:
This is based on MP-Config, but I can certainly change it.

https://github.com/jmesnil/wildfly/blob/babea3b0f925fab044a3e9a04bfe30788f9b58c5/microprofile/config-smallrye/src/main/java/org/wildfly/extension/microprofile/config/smallrye/deployment/DependencyProcessor.java#L40